packages/backend/src/graphql/project/queries/projectsQueryField.test.ts

```diff
@@ -161,4 +178,30 @@ describe('members', () => {
       ],
     })
   })
+
+  it('should return projects of members when the user is an admin', async () => {
```

Comment:
```suggestion
  it('should return projects of a user with role admin', async () => {
```

Reply:
The requesting user must be an admin to see a member's projects. Modified to:
'should return projects when the requesting user has admin role'